miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditServiceRecordController.java

```diff
@@ -148,8 +142,8 @@ public ModelAndView newServiceRecord(@PathVariable(value = "instrumentId") Long
     return new ModelAndView("/pages/editServiceRecord.jsp", model);
   }
   
-  @RequestMapping(method = RequestMethod.POST)
-  public String processSubmit(@ModelAttribute("serviceRecord") ServiceRecord record, ModelMap model, SessionStatus session)
+  @PostMapping
+  public ModelAndView processSubmit(@ModelAttribute("serviceRecord") ServiceRecord record, ModelMap model, SessionStatus session)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Replace this persistent entity with a simple POJO or DTO object. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4684)